src/templates/coveringLetter/CoveringLetter.test.tsx

```diff
@@ -0,0 +1,12 @@
+import { render, screen } from "@testing-library/react";
+import React from "react";
+import { CoveringLetterTemplate } from "./CoveringLetter";
+import { coveringLetter } from "./sample";
+
+describe("covering Letter", () => {
+  it("should render the cover letter correctly", () => {
+    render(<CoveringLetterTemplate document={coveringLetter} handleObfuscation={() => {}} />);
+
+    expect(screen.getAllByText("Documents Bundle")).toHaveLength(1);
```

Comment:
Are your sure this test is useful?

Reply:
i needed a simple test to test the rendering of the component and the CI needs at least one test so that it can pass on the test part.